### ISSUE TITLE
update failed

### ISSUE THREAD

**Issue by webservicebe** [rcschaff82/cwp_2fa#8]
Last night there was a change in 
/usr/local/cwpsrv/var/services/users/login/index.php

However because of the chattr in watch.sh the new index.php wasn't loaded and clients couldn't login (the api was changed)
Quick solution is to set chattr +i index.php.
Maybe a better solution is to put login.php as the first index on  
/usr/local/cwpsrv/conf.d/users.conf
Then only the theme has to be updated on changes.

**Comment by rcschaff82** [comment]
Do you mean that index.php was deleted, or was not moved to a different name?

**Comment by rcschaff82** [comment]
No Response.  CLosing issue.  Reopen if issue persists

**Comment by webservicebe** [comment]
** I've only responded by mail

I mean that abcdefg.php (the index of CWP) was not updated.
There was still an old version of abcdefg.php there, which was not compatible with the
new api.

So I downloaded static.cdn-cwp.com/files/cwp/el7/cwp-services.zip
wanted to copy index.php from the zip to
/usr/local/cwpsrv/var/services/users/login when I got the error:
permission denied.

After chattr -i index.php, I could copy the file.


Afterwards I copied index.pho to abdcdefg.php and login.php to index.php
